File: Backend/src/main/java/com/vidasalud/backend/model/RegistroSueno.java
package com.vidasalud.backend.model;

import jakarta.persistence.*;
import lombok.Data;
import java.time.LocalDate;

@Entity
@Data
public class RegistroSueno {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private Long usuarioId; // Relación simple con el usuario
    private LocalDate fecha;
    private Float horas;
}

File: App_VidaSalud_Grupo11-Proyecto/app/src/main/java/com/example/app_vidasalud_grupo11/model/RegistroSueno.kt
package com.example.app_vidasalud_grupo11.model

data class RegistroSueno(
    val id: Long? = null,
    val usuarioId: Long,
    val fecha: String? = null, // Usaremos String para simplificar la fecha (formato "yyyy-MM-dd")
    val horas: Float
)